### ISSUE TITLE
Does not understand deployment_minimum_healthy_percent from aws_ecs_service

### ISSUE THREAD

**Issue by RomanSaveljev** [terraform-providers/terraform-provider-aws#6361]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
Terraform v0.11.10
+ provider.aws v1.42.0
+ provider.template v1.0.0
```

### Affected Resource(s)

* aws_ecs_service

### Terraform Configuration Files

```hcl
provider "aws" {
  region      = "eu-west-1"
  max_retries = "10"
}

resource "aws_ecs_task_definition" "test" {
  family                = "test"
  container_definitions = <<EOF
        [
            {
                "dnsSearchDomains": null,
                "entryPoint": null,
                "command": [
                    "sleep",
                    "10"
                ],
                "linuxParameters": null,
                "cpu": 0,
                "environment": [],
                "ulimits": null,
                "dnsServers": null,
                "workingDirectory": null,
                "dockerSecurityOptions": null,
                "memory": 512,
                "memoryReservation": null,
                "volumesFrom": [],
                "image": "debian:stretch",
                "disableNetworking": null,
                "essential": true,
                "links": null,
                "hostname": null,
                "extraHosts": null,
                "user": null,
                "readonlyRootFilesystem": null,
                "dockerLabels": null,
                "privileged": null,
                "name": "test"
            }
        ]
    EOF
}

resource "aws_ecs_service" "test" {
  name                               = "test"
  cluster                            = "raas"
  desired_count                      = 1
  deployment_minimum_healthy_percent = 0
  deployment_maximum_percent         = 100
  task_definition                    = "${aws_ecs_task_definition.test.arn}"
}
```

### Debug Output

https://gist.github.com/RomanSaveljev/c25994922470404763d537dc2aa62f45

### Expected Behavior

Terraform succeeds creating the service

```
Terraform will perform the following actions:

  + aws_ecs_service.test
      id:                                 <computed>
      cluster:                            "raas"
      deployment_maximum_percent:         "100"
      deployment_minimum_healthy_percent: "0"
      desired_count:                      "1"
      iam_role:                           <computed>
      launch_type:                        "EC2"
      name:                               "test"
      scheduling_strategy:                "REPLICA"
      task_definition:                    "arn:aws:ecs:eu-west-1:853142832404:task-definition/test:1"
```

### Actual Behavior

Terraform fails to create the service, because it does not pick up `deployment_minimum_healthy_percent` set to `0`. Note that setting it to `1` makes it work, but this is not the desired behavior.

```
Terraform will perform the following actions:

  + aws_ecs_service.test
      id:                         <computed>
      cluster:                    "raas"
      deployment_maximum_percent: "100"
      desired_count:              "1"
      iam_role:                   <computed>
      launch_type:                "EC2"
      name:                       "test"
      scheduling_strategy:        "REPLICA"
      task_definition:            "${aws_ecs_task_definition.test.arn}"
```

The issue can be worked around by downgrading AWS provider plugin to **v1.41.0**.

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by bflad** [comment]
Hi @RomanSaveljev 👋 Thanks for the report and very sorry for the trouble! We actually have a fix for this merged (#6316 / [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md)), which will go out in the next two days. Hope this helps.

**Comment by RomanSaveljev** [comment]
Thanks! That issue look like exact duplicate. Sorry that I did not search for it enough

**Comment by bflad** [comment]
This has been released in [version 1.43.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1430-november-07-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
